generate/gogen/types.go

```diff
@@ -186,7 +188,6 @@ func (g *generator) convertType(juteType parser.Type) (*goType, error) {
 		}
 		return &goType{
 			typeID:    typeClass,
-			isPtr:     true,
 			classType: prefix + t.ClassName,
 		}, nil
 	}
```

Comment:
I think the below comment can be dropped

> 	// TODO(bbennett): Since we always use pointers for class references we
> 	// don't really support external classes used as keys. Do we need this?

Reply:
Removed